docker/Dockerfile

```diff
@@ -13,11 +13,11 @@ RUN apt-get update && \
     pip3 install --upgrade pip && \
     pip3 install setuptools==58.5.3
 
-RUN pip3 install torch -f https://download.pytorch.org/whl/torch_stable.html
+RUN pip3 install torch==1.13.1 torchvision==0.14.1 torchaudio==0.13.1 -f https://download.pytorch.org/whl/torch_stable.html
```

Comment:
Install torch from cpu channel. See the [official docs](https://pytorch.org/get-started/locally/) for more details. Torch and scatter are installed after this pip install too. Have a look out for redundant libraries or libraries attempted to install twice.

Reply:
Also don't pin the version of torch here.

Reply:
But what about [this discussion](https://discuss.pytorch.org/t/libtorch-cuda-so-is-too-large-2gb/103155/24) on PyTorch's community? I have resolved it and unpinned the version